README.md

```diff
@@ -93,6 +93,9 @@ Key | Description | Default value
 | run-before | Script that is executed before the actual command | *no default*
 | run-after | Script that is executed after the actual command | *no default*
 | alias | Allows to set short aliases for long commands<br>`my_command: "--ri --with-docker-mount --image=my-image --image-version=my-tag -E my-script.py"` | *no default*
+| auto-update | Toggles the auto update check. Will only perform the update after the delay | true
+| auto-update-delay | Delay before running auto-update again  | 2h (2 hours)
```

Comment:
I like when users know which format to use and not just guess. Here, It seems to be the "duration" format of Go: https://golang.org/pkg/time/#ParseDuration

Reply:
It is the duration, I was following this example (line 86):
```
| docker-refresh | Delay before checking if a newer version of the docker image is available | 1h
```